circle.yml

```diff
@@ -1,10 +1,16 @@
-machine:
-  python:
-    version: 3.5.2
-  environment:
-    ANSIBLE_REMOTE_USER: ubuntu
-    PYTHON27: /opt/circleci/python/2.7.12/bin/python
-    TEST_HOSTS: integration/hosts
-test:
-  override:
-    - cd tests && make test
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.5.4
+        environment:
+            TEST_HOSTS: integration/hosts
```

Comment:
@smarnach I removed all except this one, cause it's needed in another command too. See: https://github.com/open-craft/ansible-load-balancer/blob/master/tests/Makefile#L26

Reply:
@rocioar I'm sorry, this is still not what I meant.  What I meant was something very simple – just move the enviroment variables to the command _in this file_, so that the command looks like this:

    - run:
        name: Run tests
        command: cd tests && make test
        environment:
           ￼ANSIBLE_HOST_KEY_CHECKING: false
           ANSIBLE_REMOTE_USER: ubuntu
           TEST_HOSTS: integration/hosts

It wasn't a very important comment – I just felt the enviroment variables belong to the test command rather than the image, and I'm sorry I didn't make this clear right away.  It always feels so obvious to _myself_ what I mean that I forget to properly explain it. :)

Reply:
oops @smarnach this took me long to understand, when you said the test command I thought you meant the Makefile command... sorry. I just fixed it

Reply:
I really didn't make it clear – what you did was perfectly reasonable. :)